bin/genconfig2

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib_dir)
 
 require 'beaker-hostgenerator'
 
-STDERR.puts("Warning: 'genconfig2' is deprecated and will be removed in beaker-hostgenerator 1.x\n\n")
+$stderr.puts("Warning: 'genconfig2' is deprecated and will be removed in beaker-hostgenerator 1.x\n\n")
```

Comment:
Haha yeah my bad, I realized afterwards that we probably don't have any references to `$stdout` since they'd be hidden behind the calls to `puts`.